package.json

```diff
@@ -45,10 +45,10 @@
     "gulp-sort": "^2.0.0",
     "gulp-transform": "^1.1.0",
     "gulp-typescript": "^3.1.5",
-    "gulp-uglify-harmony": "https://github.com/jiborobot/gulp-uglify-harmony.git#master",
+    "gulp-uglify-harmony": "git+https://github.com/jiborobot/gulp-uglify-harmony.git#master",
     "mocha": "^3.2.0",
     "rimraf": "^2.6.1",
-    "typescript": "^2.2.1",
+    "typescript": "2.3.*",
```

Comment:
Almost :) `*` means *any* version. Make it `~2.3.4`, which means only *patch* number (last one) can increment.